src/lib/core/ripple/_ripple.scss

```diff
@@ -10,12 +10,18 @@ $mat-ripple-color-opacity: 0.1;
   .mat-ripple {
     overflow: hidden;
 
+    // Limit style recalculations to the ripple container, as much as possible.
+    contain: strict;
```

Comment:
I would consider this a breaking change because people could have changed the `overflow` or might do something else with the ripple container.

Since `matRipple` is a directive that can be applied on any element, and is not necessarily applied to an additional element, this might break some people.

Reply:
I see what you mean, but I think that it would be a very narrow (if any) slice of users since they'd also have to have overridden the `overflow: hidden` at which point the ripple isn't super usable.

Reply:
That's true. I don't mind seeing how it goes because technically the ripples are not even documented yet. Though if we fully follow the semver concept, this would be breaking.

Reply:
http://www.hyrumslaw.com/

In reality, _any_ change is a breaking change-- we're always making a judgement call to some extent.

Reply:
Kind of true, that's why I agreed that it's more unlikely that people might run into issues with that _particular_ change.